test/extended/util/cli.go

```diff
@@ -85,21 +85,27 @@ func NewCLI(project, adminConfigPath string) *CLI {
 
 	g.BeforeEach(client.SetupProject)
 
-	ProwGCPSetup(client)
-
 	return client
 }
 
 // ProwGCPSetup makes sure certain required env vars are available in the case
 // that extended tests are invoked directly via calls to ginkgo/extended.test
 func ProwGCPSetup(oc *CLI) {
```

Comment:
We need to doc this, totally new to me ...

Reply:
Did you specific location in mind for this doc?  ... I might have a few cycles to lend some verbiage.

As part of the trial/error discovery I went through in getting the jenkins ext tests to run via the direct ginkgo invocation vs. the test/extended/core.sh path, I ultimately ran into https://github.com/openshift/origin/blob/master/hack/lib/util/environment.sh

There is quite the list of env's needed to start a openshift server ... the presence/assumption of these env's existing as a result of starting an openshift server specifically for running the tests undoubtedly crept into the test implementations.

Reply:
I don't really get why we're calling ginkgo directly, need to ask Clayton